library/src/main/java/com/liferay/mobile/push/PushNotificationsReceiver.java

```diff
@@ -23,14 +24,22 @@
  * @author Bruno Farache
  */
 public abstract class PushNotificationsReceiver
-	extends WakefulBroadcastReceiver {
+	extends BroadcastReceiver {
 
+	public static int JOB_ID = 10001;
```

Comment:
why this number? what if there's some collision with user ids?

Reply:
It could be any, if it collides with other job id, users can change it